### PR TITLE
Add .sub_label

### PR DESCRIPTION
**Pull request by jrubenoff**
Fixes dobtco/screendoor-v2#1228

### CODE REVIEW

File: vendor/assets/stylesheets/dvl/core/forms.scss [hunk 1]
```diff
@@ -153,6 +153,11 @@ label {
   cursor: pointer;
   font-weight: 600;
   font-size: 0.9rem;
+  &.sub_label {
```
Comment (dobthound) [inline vendor/assets/stylesheets/dvl/core/forms.scss]: Selector `sub_label` should be written in lowercase with hyphens

**Comment by ajb** [comment]
Can you add some markup to the `forms.rb` page, too?

File: vendor/assets/stylesheets/dvl/core/forms.scss (hunk 1 above)
Comment (ajb) [inline vendor/assets/stylesheets/dvl/core/forms.scss]: Whoops, I might have to redeploy Hound...

**Comment by jrubenoff** [comment]
Done

**Comment by ajb** [comment]
Sorry, I coulda just waited for the associated PRs. Looks great, I'll
review in a bit

On Mon, Jan 12, 2015 at 4:02 PM, Josh Rubenoff <notifications@github.com>
wrote:

> Done
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/dvl-core/pull/32#issuecomment-69645239>.
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker

**Comment by ajb** [comment]
This PR looks great, but it's not going to be entirely necessary, since formrenderer-base has now been changed to use the `.fr_sub_label` class. I know the dependency stuff can get confusing, but our dependency graph looks like this:

img

So theoretically, `dvl-core` should not even know about `formrenderer-base`. We can certainly merge this PR since the styles look handy, but we'll still need to add the styles to [Screendoor's formrenderer overrides](https://github.com/dobtco/screendoor-v2/blob/9b63a3321cb80c65ea9a86dfa9f8323136959926/app/assets/stylesheets/components/form_renderer.scss).

**Comment by jrubenoff** [comment]
Let's just ditch this PR, then.

**Pull request closed without merging**